### DISCUSSION TITLE
difference between Hobby and Pro version inbresponse time and handling traffic aspects?

### DISCUSSION THREAD

**Discussion by Ebdulmomen1** [vercel/vercel#5491] [Help] [answered]
Hi,i have a website deployed on a Hobby plan and i experience a little bit of inconsistency in terms of response time from the server, sometimes it is alright but there are other times that it is just not as responsive, is there a difference between the Hobby and Pro version in terms of being able to handle more amount of requests(traffic) and is it going to improve the response time if i upgrade?

**Comment by cobrexus** [comment, marked as answer]
That's due to something called a [cold start](https://en.wikipedia.org/wiki/Cold_start_(computing)), and upgrading can't remove that. It's something that serverless functions will suffer from, because they open a connection and then close it, as opposed to serverful which is running all the time.

In case you aren't sure about what the pro plan offers, according to [Vercel's pricing](https://vercel.com/pricing), the pro plan includes the following additional features:

![image](https://user-images.githubusercontent.com/34424160/100719225-41713480-33b4-11eb-8c8e-5ecc36b392a1.png)

It looks like all the main functionality in deployment and serverless functions are included in the hobby plan.
